### DISCUSSION TITLE
Permanently banned on dota 2

### DISCUSSION THREAD

**Discussion by Originallyaizen** [ValveSoftware/Dota2-Gameplay#4607] [Q&A]
so my account has a permanent ban on dota 2 for no reason i mean i have been playin this game for the past 10 years my acc is 12 years old has 27k hours on this and still im perma banned without any explaination im not a cheater or script user im not scamming anyone and certainly not a smurf so why is my account banned can someone please tell me?

**Comment by trichome3** [comment]
According to unconfirmed sources, crashing dedicated servers while playing a matchmaking game can also lead to a permanent VAC ban. I am sure the website may have something to tell on that[.](https://sellerdash.pk/) I hope so.

**Reply by kylogenetictree** [reply]
But that's through unconfirmed sources[.](https://sellerdash.pk/) Had it been an official, recognized term, the official platform of Dota would have already mentioned it.
